### DISCUSSION TITLE
Connecting multiple vehicles via QGC

### DISCUSSION THREAD

**Discussion by andreamanera** [RosettaDrone/rosettadrone#168] [General]
Hello, I am working on this project and I was wondering if, using Rosetta and QGC it could be possible to connect multiple drones to the same GCS, obviously using a smartphone and a radio control for each drone that I want to connect to the GCS, do you know if this feature has been implemented? If yes could you give me some advices about it? and if no do you think that it could be possible to implement it by myself?

**Comment by andreamanera** [comment]
Okay I'm sorry but we were able to solve the problem and correctly connect two vehicles, thank you

**Comment by kripper** [comment]
I didn't test this setup so there could be some issues, but in general multiple vehicules support is feasible.

**Comment by The1only** [comment]
Did you add the Mavlink ID in the config? 
Or how was it solved?

**Reply by andreamanera** [reply]
Yes, setting up different Mavlink ID for each drone directly from QGC it's easy to connect to Rosetta and everything works fine, also waypoint missions, tested with 2 DJI mini 2 and 2 DJI RC-N1.

**Reply by ricardohalfeld** [reply]
Are you working without video transmission or did you solve the issues with DJI Mini 2 streaming?

**Comment by kripper** [comment]
@andreamanera please make sure to pull latest GIT changes since I fixed the PID parameters recently (for smoother flying).
